examples/adapter-tests/src/context.js

```diff
@@ -31,6 +31,6 @@ module.exports = function context(fractal) {
 
     it('loads fully rendered component from another component context key', async () => {
         const ctx = await fractal.components.find('@reference-full').getResolvedContext();
-        expect(ctx.reference).toBe('<p>JS Context</p>\n');
+        expect(ctx.reference).toMatchSnapshot();
```

Comment:
changed some of these tests to snapshots because it's a bit hard to enforce exactly the same output for all adapters - for example, React render result is not prettified and usually has a `data-reactroot` attribute, etc.